docs/identities/get-started/account-recovery.mdx

```diff
@@ -44,6 +44,25 @@ configure account recovery in your Ory project.
 
 ### Enable SMS account recovery
 
+This is enabled by turning a feature flag to 'on' at the project level (opt-in). This flag is enabled by default for newly created
+projects.
+
+This feature only works when the recovery strategy is `code` (not `link`).
+
+#### As a user
+
+If you only have an email address and no phone number, there is no perceivable difference.
+
+If you do have a phone number and want to receive a SMS containing the recovery code, make sure that this phone number is
+registered as a recovery address.
+
+If you have multiple recovery addresses registered, during the recovery flow, you will be presented a list of your (masked)
+addresses and you will be prompted to choose one of them.
+
+You may also be prompted to enter this address in full to prove that you know the complete address, for security reasons.
```

Comment:
what means "may" here? is the user prompted to provide the full address or not @gaultier

Reply:
So basically if the user inputs their address in step 1 and then chooses that address in step 3 they wont be asked to input the full address bc they did already, that makes sense. 

how about 
```suggestion
To confirm ownership, the user may be asked to enter the complete address after selecting it. This verification step is skipped if they already entered that same address to begin the recovery process.
```